vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -75,20 +82,20 @@ mkdir testBundle; cd testBundle
 ```
 
 ### Run the Bundler
-1. Create a child directory to hold the bundler output. Using the name `bundle` allows you to easily use the `ent prj` command with this bundle.
+1. Create a child directory to store the bundler output. Choosing the name `bundle` allows you to easily use the `ent prj` command with this bundle.
 ``` sh
 mkdir bundle
 ```
-2. Run the bundler command with your preferred settings. 
+2. Run the bundler command with your preferred settings:
```

Comment:
is preferred settings a bit vague? wondering which settings are variable.

If the command below had caps for the settings that need to be edited, that would make this unnecessary

Reply:
pretty sure this just means you have the freedom to modify command parameters. no different from how we say "modify as needed"

Reply:
I think that freedom is implied by our use of "Your" everywhere. So this can just be "Run the bundler command" or even "Run the bundler"